src/zeta/aca_zeta_programming.cpp

```diff
@@ -184,6 +137,8 @@ int ACA_Zeta_Programming::_delete_zeta_group_entry(zeta_config *zeta_cfg)
   return overall_rc;
 }
 
+// Determine whether the group table rule already exists?
+// I am not sure if this approach is appropriate?
```

Comment:
it is a decent approach. Need to think about the case that the group rule is there but it is the exactly as the one we expected, like the bucket destination is wrong. That's an improvement to consider in the future and not a blocker now.